lxd/db/cluster/permissions.go

```diff
@@ -33,3 +40,73 @@ type PermissionFilter struct {
 	EntityType  *EntityType
 	EntityID    *int
 }
+
+// GetPermissionEntityURLs accepts a slice of Permission and returns a map of entity.Type, to entity ID, to api.URL.
+// The returned map can be keyed on Permission.EntityType and Permission.EntityID to get a URL for the permission (e.g.
```

Comment:
"can be" - how else can it be represented?

Reply:
"should be" - it's a bit of a weird comment looking back on it. I was trying to make clear the intended usage.